decidim-dev/lib/decidim/dev/component.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+Decidim.register_component(:dummy2) do |component|
```

Comment:
I'm not sure what's the need for adding `dummy2`. Why do we need it?

Reply:
I just realised that we don't ... fixed in [4e8f956](https://github.com/decidim/decidim/pull/13237/commits/4e8f956f8c78e84fe6de8a54e76c12761991ece4)